var/spack/repos/builtin/packages/python/package.py

```diff
@@ -56,7 +61,6 @@ class Python(Package):
     version(
```

Comment:
Can you wrap this back onto a single line?

Reply:
Does that change the hash? If yes, then rather not :p

Reply:
It should not change the hash.

Reply:
done